torch/jit/_script.py

```diff
@@ -36,6 +36,14 @@
 from torch.overrides import (
     has_torch_function, has_torch_function_unary, has_torch_function_variadic)
 
+_IS_MONKEYTYPE_INSTALLED = True
+try:
+    from monkeytype import trace as monkeytype_trace  # type: ignore[import]
+    from torch.jit._monkey_config_jit import JitTypeTraceConfig, type_trace_db
+    monkeytype_config = JitTypeTraceConfig(type_trace_db)
```

Comment:
This is related to:
https://github.com/pytorch/pytorch/pull/54513/files#r610384224

I think each `torch.jit.script` call should maintain its own trace database, sharing it among multiple calls can lead to confusing behaviors.

Reply:
Agreed and fixed